src/database/migrations/1730988922695-create-user.ts

```diff
@@ -4,12 +4,16 @@ export class CreateUser1730919143524 implements MigrationInterface {
   name = 'CreateUser1730919143524';
 
   public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TYPE "public"."user_role_enum" AS ENUM('admin', 'user')`,
+    );
```

Comment:
I recommend creating a table to store roles and connecting it to the users table. It would be good for scalability. Optional though.

Reply:
we don't have much roles. Requirements say only user role is enough for now